### PR TITLE
fix lab list

### PR DESCRIPTION
**Pull request by dimitri-yatsenko**

### CODE REVIEW

File: contents/intro/05-who.rst [hunk 1]
```diff
@@ -3,21 +3,23 @@
 Who uses DataJoint
 ==================
 
-DataJoint was originally developed by working systems neuroscientists at Andreas Tolias' Lab at Baylor College of Medicine. 
-
+DataJoint was originally developed by working systems neuroscientists at Andreas Tolias' Lab at Baylor College of Medicine
```
Comment (austin-hilberg) [inline contents/intro/05-who.rst:6]: I think technically it should be "Tolias's Lab", because it isn't plural.

**Review by eywalker — APPROVED** [review]

File: contents/intro/05-who.rst (hunk 1 above)
Comment (austin-hilberg) [reply contents/intro/05-who.rst:6]: https://apvschicago.com/2011/06/apostrophe-s-vs-apostrophe-forming.html

Comment (austin-hilberg) [reply contents/intro/05-who.rst:6]: Let's leave it as is.

Comment (eywalker) [reply contents/intro/05-who.rst:6]: Changing it to `Tolias Lab (Andreas Tolias)`

**Pull request merged**